source/Session.h

```diff
@@ -156,6 +156,12 @@ class SessionConfig : public FpsConfig {
 	Any toAny(const bool forceAll = false) const;
 	float getTrialsPerBlock(void) const;			// Get the total number of trials in this session
 	Array<String> getUniqueTargetIds() const;
+
+	void respawnPlayer() { player.respawnToPos = true; }
+	void propagatePlayerControlsToAllFromGUI() { player.propagatePlayerConfigsToAll = true; }
```

Comment:
I think all of these methods should just be one `propogatePlayerConfig()` that takes a `PlayerConfig`. You can easily have the `broadcast` and `fromFile` fields be flags for this method, in fact it seems like you are already doing this to some extent here.

Reply:
I was not able to find ways to send values to a function through the GUI. That's why I wrote these.

Reply:
This makes sense, I'm okay with having a 2nd special call that makes this avoid parameters. If possible can we make this call the general purpose method w/ the correct parameter values?

Reply:
What do you mean by calling a general purpose method with parameters? I have not been able to find any easy way to call parameterized method from the gui unfortunately.